### PR TITLE
feat(angular): update ngrx to 10.0.0

### LINKED ISSUE #3542: Update ngrx to v10
## Description
Ngrx has been updated to v10 a few days ago. Its a good time to update the versions file and add a migration
## Motivation
Ngrx is updated and aligned with angular / nx release

### PR DESCRIPTION
**Pull request by mehrad-rafigh**
ISSUES CLOSED: #3542

## Current Behavior
Ngrx is currently at v9

## Expected Behavior
Ngrx is updated to v10

## Related Issue(s)
#3542

Fixes# #3542

### CODE REVIEW

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Will you update the `10.1.0` migration references to `10.2.0`?

**Comment by mehrad-rafigh** [comment]
Hi @brandonroberts sure will do and update the PR tonight

**Comment by mehrad-rafigh** [comment]
Hi @brandonroberts 
Did the changes you requested

File: packages/angular/migrations.json [hunk 1]
```diff
@@ -30,6 +30,11 @@
       "version": "10.0.0-beta.1",
       "description": "Upgrades Angular and Angular CLI to 10.0.0",
       "factory": "./src/migrations/update-10-0-0/update-10-0-0"
+    },
+    "update-10-2-0": {
+      "version": "10.2.0-beta.1",
+      "description": "Upgrade Ngrx to 10.0.0",
```
Comment (brandonroberts) [inline packages/angular/migrations.json:36]: nit: casing

```suggestion
      "description": "Upgrade NgRx libraries to version 10.0.0 ",
```

Comment (mehrad-rafigh) [reply packages/angular/migrations.json]: Done!

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**